src/utils/CopyToClipboard/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Icon } from '../..';
+import copyToClipboard from './copyToClipboard';
+
+const StyledButton = styled.button`
+  background: none;
+  color: inherit;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+`;
+
+type Props = {
+  textToCopy: string;
+  className?: string;
+};
+
+const CopyToClipboard = ({
```

Comment:
`CopyBtn`? feels weird to have `copyToClipboard` and `CopyToClipboard` in the same file

Reply:
I think CopyToClipboard is much explicit than just CopyBtn. I renamed the function instead.

Reply:
Explicit for what? What is CopyToClipboard component? A button? A text which would be copied if you click on it?

Reply:
@mikheevm renamed.